apps/backend/tests/song.test.ts

```
@@ -27,6 +36,115 @@ describe("Song E2E Tests", () => {
+			const role = await prisma.artistRole.create({ data: { role: "composer" } });
+			roleId = role.id;
+		});
+
+		test("should create a song with lyrics", async () => {
+			const { data, status } = await api.v2.song.post({
+				name: "Song With Lyrics",
+				lyrics: [{ language: "zh", plainText: "歌词内容" }],
+			});
+
+			expect(status).toBe(201);
+			const lyricsRows = await prisma.lyrics.findMany({ where: { songId: (data as any).id } });
```

Biome (2.4.10)

[error] Unexpected any. Specify a different type.

(lint/suspicious/noExplicitAny)